apps/settings/elements/about_browser_os_description.html

```diff
@@ -0,0 +1,17 @@
+<element name="about-browser-os-description" extends="section">
```

Comment:
did you use $ git mv command in order to rename this file? Technically speaking this file is the same as apps/settings/elements/about_firefox_os_description.html, but witha  few minor changes in namings.